src/codegen/options.ts

```diff
@@ -1,14 +1,16 @@
-import { TestOptions, Threshold } from '@/types/testOptions'
+import { LoadZoneData, TestOptions, Threshold } from '@/types/testOptions'
 import { stringify } from './codegen.utils'
 import { omit } from 'lodash-es'
 
 export function generateOptions({
   loadProfile,
   thresholds,
+  cloud,
 }: TestOptions): string {
   const options = omit(loadProfile, ['executor'])
   const data = {
     ...options,
+    ...(cloud && generateCloudOptions(cloud)),
```

Comment:
The `cloud` options are ignored locally so it's okay to always generate them. However, we still need some kind of messaging in the Load Zones tab to let users know about this.